pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -189,7 +189,12 @@ class Build {
         if (buildConfig.VARIANT == "corretto") {
             suffix="corretto/corretto-${javaNumber}"
         } else if (buildConfig.VARIANT == "openj9") {
-            suffix = "ibmruntimes/openj9-openjdk-jdk${javaNumber}"
+            def openj9JavaToBuild = buildConfig.JAVA_TO_BUILD
+            if (openj9JavaToBuild.endsWith("u")) {
+                // OpenJ9 extensions repo does not use the "u" suffix
+                openj9JavaToBuild=openj9JavaToBuild.substring(0,openj9JavaToBuild.length()-1)
```

Comment:
Could this not use `${javaNumber}` as it did previously or will the substring return somethign different in some cases?

Reply:
That was the problem javaNumber is 17 for JAVA_TO_BUILD=jdk, if your building jdk you want the jdk extensions repo